src/components/Section.js

```diff
@@ -14,4 +13,15 @@ export default class Section {
   addItems(element) {
     this._container.prepend(element);
   }
+
+  clearItems() {
+    this._container.innerHTML = ''; 
+  }
+
+  renderInitialItems(items) {
```

Comment:
Please, use `renderItems` not to duplicate the same code. Or delete `renderItems` from your class